rel-eng/tito.props

```diff
@@ -434,26 +434,31 @@ whitelist = ansiblerole-insights-client
 disttag = .el7
 whitelist = foreman-release
   katello-host-tools
+  katello-repos 
```

Comment:
Why do we host katello-repos in the client repository? I'd understand hosting this in the foreman main (/server) repository.

Reply:
The katello-repos spec also builds the katello-client-repos rpm 

https://github.com/theforeman/foreman-packaging/blob/2843207f7c6bfe60bc256a12a02e047aee1a3f7a/packages/katello/katello-repos/katello-repos.spec#L44

Reply:
https://github.com/theforeman/foreman-packaging/pull/3113 removes it and I'd prefer to rebase on that.

Reply:
I can rebase on that when its merged